modules/cryoGridTechnical/iSaveState.m

```diff
@@ -1,3 +1,11 @@
-function iSaveState( fname, T, wc, t, SEB, PARA, GRID)
-  save( fname, 'T', 'wc', 't', 'SEB', 'PARA', 'GRID' );
+function iSaveState( fname, T, wc, t, SEB, PARA, GRID)

```

Comment:
We should implement a function which loads the state from a saved state.

Reply:
Yes ! Would be nice for debugging also !